python/tchannel/thrift/transport/tornado_base.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) 2015 Uber Technologies, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+from __future__ import absolute_import
+
+from thrift import Thrift
+
+from tornado import gen
+from tornado import ioloop
+try:
+    from tornado.queues import Queue  # Included in Tornado 4.2
+except ImportError:
+    from toro import Queue
+
+from tchannel.messages.common import Types
+from .base import TChannelTransportBase
+
+
+class TChannelTornadoTransportBase(TChannelTransportBase):
+    """A Thrift Transport to send requests over an open TChannel connection.
+
+    Works with Thrift clients generated with the ``tornado`` option enabled.
+
+    .. code-block::
+
+        transport = TChannelTornadoTransport(
```

Comment:
Move the example back to the implementation. It's specific to that implementation.